msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -317,9 +317,12 @@ internal Authority GetAuthority(IAccount account)
             }
 
             var handler = new SilentRequest(
-                    HttpManager, CryptographyManager, CreateRequestParameters(authority, scopes, account, UserTokenCache),
-                forceRefresh)
-            { ApiId = apiId };
```

Comment:
the intiailization of ApiId is now missing?

Reply:
It was moved in ctor, since it is pretty much required for all flows.